.github/workflows/cd_release.yml

```diff
@@ -297,7 +297,7 @@ jobs:
       with:
         user: __token__
         password: ${{ secrets.PyPI_token }}
-        repository_url: https://test.pypi.org/legacy/
+        repository-url: https://test.pypi.org/legacy/
```

Comment:
change due a bug or a version change?

Reply:
Version change in the used action. I can put a comment in the final merging commit message 👍

Reply:
done